extension/chrome/texts/email_provider_auth_security.htm

```diff
@@ -1,12 +1,33 @@
 <!-- © 2016-2018 FlowCrypt Limited. Limitations apply. Contact human@flowcrypt.com -->
 
-<div class="lightboxed">
-  <p>To work properly, your FlowCrypt browser extension needs to connect to your email provider (Gmail or Outlook).
-    This allows you to encrypt and decrypt messages when you choose to.</p>
-  <p>The plugin will receive so called API Tokens. These tokens, and all related privileges (to send and read messages)
-    will strictly stay within your own browser, stored locally in your computer for maximum security.</p>
-  <p><b>By design, the tokens cannot and will never be exposed to FlowCrypt servers or its developers.</b></p>
-  <p>The code running FlowCrypt is publicly available and anyone can verify the underlying security. Visit <a href="https://github.com/FlowCrypt/flowcrypt-browser"
-      target="_blank">FlowCrypt Sources Repository</a> to learn more or inspect the code. Alternatively, please ask us
-    directly at human@flowcrypt.com</p>
-</div>
\ No newline at end of file
+<html>
+
+<head>
+  <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+  <title>FlowCrypt</title>
+  <link rel="stylesheet" href="/css/cryptup.css">
+  <link rel="stylesheet" href="/css/sweetalert2.css">
+  <link rel="stylesheet" href="/css/settings.css">
+  <link rel="stylesheet" href="/lib/bootstrap/bootstrap.min.css">
+  <link rel="stylesheet" href="/css/animate.css">
+  <link rel="stylesheet" href="/css/open-sans.css">
+  <link rel="stylesheet" href="/css/mobile-menu-styling.css">
+  <link rel="icon" href="/img/logo/flowcrypt-logo-19-19.png">
+</head>
+
+<body style="display: none;">
+  <div class="line">To work properly, your FlowCrypt browser extension needs to connect to your email provider (Gmail or Outlook).
+    This allows you to encrypt and decrypt messages when you choose to.</div>
+  <div class="line">The plugin will receive so called API Tokens. These tokens, and all related privileges (to send and read messages)
+    will strictly stay within your own browser, stored locally in your computer for maximum security.</div>
+  <div class="line"><b>By design, the tokens cannot and will never be exposed to FlowCrypt servers or its developers.</b></div>
+  <div class="line">The code running FlowCrypt is publicly available and anyone can verify the underlying security. Visit <a href="https://github.com/FlowCrypt/flowcrypt-browser"
+      target="_blank">FlowCrypt Sources Repository</a> to learn more or inspect the code. Alternatively, please <span class="ask_support_assistance_container">ask us directly at human@flowcrypt.com</span>.</div>
```

Comment:
Let's make `ask_support_assistance_container` emtpy

Reply:
understand sir... I won't be worrying about default values from now on since I realized extensions won't work without javascript. thank you